Plugins/GitUIPluginInterfaces/CommitData.cs

```diff
@@ -35,5 +35,10 @@ public sealed class CommitData
         /// Gets and sets the commit message.
         /// </summary>
         public string Body { get; set; }
+
+        /// <summary>
+        /// Notes. Really.
```

Comment:
git-notes is not known by all
```suggestion
        /// Git Notes https://git-scm.com/docs/git-notes
```